### ISSUE TITLE
Adding a column

### ISSUE THREAD

**Issue by iksub** [evrenvarol/outlook-taskboard#29]
Hello,
I have manipulated the files and added another column. It appeared in the lineup and button +, however it is missing the count and does not display items which are dropped there.
How can I make this column work to display items like any other column?

**Comment by BillyMcSkintos** [comment]
Would need to see the config.js and kanban.html.

**Comment by iksub** [comment]
[kanban - Copy.zip](https://github.com/evrenvarol/outlook-taskboard/files/1171286/kanban.-.Copy.zip)

Attached here.
Please note: I am not a developer, I am just going by association.
Also, please note that I was able to get to the custom properties but in a weird way. I need both lines for custom properties, for one of them to show. Right now it shows only substatus. Maybe you can look into that as well? 
Much appreciated!

**Comment by vaz399** [comment]
Hi iskub,
I looked at your archive. You added a new folder "CR".
You must add a task folder with the same name and change the app.js too.

**Comment by iksub** [comment]
Yes, I did that too. In all files. And added the folder. As I said, the column shows but no items which i drop there show.

Sent using iPhone

**Comment by vaz399** [comment]
@iksub 
Show me the app.js  That's what I got 
![kanban](https://user-images.githubusercontent.com/22456702/28747364-b25fbfbc-74a5-11e7-8dbc-213894f3614f.png)

**Comment by iksub** [comment]
Here is the app file:
[app.zip](https://github.com/evrenvarol/outlook-taskboard/files/1186106/app.zip)

**Comment by vaz399** [comment]
Hi @iksub,
This problem occurs because you invoke the "crTask" in the kanban.html file, but the scope name is "CRTask" in app.js.
So, you can change the name to one kind and it's to solve your problem.

**Comment by iksub** [comment]
That fixed it. Gracias!

What about the custom fields display? Any thoughts on that one?

Sent using iPhone
